draft-gpew-priv-ppm.md

```diff
@@ -378,7 +378,11 @@ in the "type" field (within the PPM URN namespace "urn:ietf:params:ppm:error:"):
 | unrecognizedMessage     | The message type for a response was incorrect or the payload was malformed. |
 | unrecognizedTask        | An endpoint received a message with an unknown task ID. |
 | outdatedConfig          | The message was generated using an outdated configuration. |
+| staleReport             | Report could not be processed because it arrived too late. |
+| reportFromTheFuture     | Report could not be processed because its timestamp is too far in the future. |
 | batchInvalid            | A collect or aggregate-share request was made with invalid batch parameters. |
+| insufficientBatchSize   | There are not enough reports in the batch interval. |
```

Comment:
Are all of these fatal?

Reply:
Yes, inasmuch as they represent conditions that the aggregators cannot recover from. `insufficientBatchSize` for instance means there aren't enough reports in `CollectReq.batch_interval` to satisfy the task's `min_batch_size`, which means the resulting aggregate would violate client privacy. The description text could be improved on this one.

Reply:
Well, but could I aggregate some more into the batch and then ask for collect?

Reply:
Yes, absolutely. Getting this error is how a collector would know to either try again with a bigger `batch_interval` or wait for more reports to arrive.

Reply:
That makes sense. And for this error, we should add some text to the section on collecting explaining what a collector could do if they encounter it.

- [ ] explain in `collect-flow` what collectors should do in the face of `insufficientBatchSize`

Reply:
@tgeoghegan can we file an issue to track this?

Reply:
Done: #239